### PR TITLE
move emoji/15.0 files to emoji/dev

### PR DESCRIPTION
**Pull request by markusicu**
... as agreed for all unreleased files, and consistent with the ucd/idna/security/... files.
The emoji files for 15 had been misplaced in PR #232.

Also sync the two data files in the repo that are still duplicated between data/emoji and data/ucd. (See issue #152.)

Also update the two emoji readmes, and
remove the emoji/dev/ReadMe-ucd-emoji.txt which was a copy of ucd/dev/emoji/ReadMe.txt.

### CODE REVIEW

File: unicodetools/data/emoji/dev/internal/emojiOrdering.txt [hunk 1]
```diff
@@ -153,31 +154,32 @@
...
 @ animal-mammal
-🐵 🐒 🦍 🦧 🐶 🐕 🦮 🐕‍🦺 🐩 🐺 🦊 🦝 🐱 🐈 🐈‍⬛ 🦁 🐯 🐅 🐆 🐴 🐎 🦄 🦓 🦌 🦬 🐮 🐂 🐃 🐄 🐷
+🐵 🐒 🦍 🦧 🐶 🐕 🦮 🐕‍🦺 🐩 🐺 🦊 🦝 🐱 🐈 🐈‍⬛ 🦁 🐯 🐅 🐆 🐴 🫎 🫏 🐎 🦄 🦓 🦌 🦬 🐮 🐂 🐃
+🐄 🐷
 🐖 🐗 🐽
 🐏 🐑 🐐 🐪 🐫 🦙 🦒 🐘 🦣 🦏 🦛 🐭 🐁 🐀 🐹 🐰 🐇 🐿️ 🦫 🦔 🦇 🐻 🐻‍❄️ 🐨 🐼 🦥 🦦 🦨 🦘 🦡
 🐾
 @ animal-bird
-🦃 🐔 🐓 🐣 🐤 🐥 🐦 🐧 🕊️ 🦅 🦆 🦢 🦉 🦤 🪶 🦩 🦚 🦜
+🦃 🐔 🐓 🐣 🐤 🐥 🐦 🐧 🕊️ 🦅 🦆 🦢 🦉 🦤 🪶 🦩 🦚 🦜 🪽 🪿 🐦‍⬛
```
Comment (nedley) [inline unicodetools/data/emoji/dev/internal/emojiOrdering.txt:163]: @markusicu Please review https://github.com/unicode-org/emoji/pull/24 and take that version of this file.

**Review by nedley — CHANGES_REQUESTED** [review]
Please review https://github.com/unicode-org/emoji/pull/24 and take that version of emojiOrdering.txt.

**Comment by markusicu** [comment]
> Please review [unicode-org/emoji#24](https://github.com/unicode-org/emoji/pull/24) and take that version of emojiOrdering.txt.

For this PR here, I would prefer to just move the existing data files into the right place, rather than also change the data, especially from a pending PR.

Also, I don't see any "ordering" file in the other PR with this name or contents.

I can rubber-stamp https://github.com/unicode-org/emoji/pull/24, if you like, but not much more; and it still has an unresolved negative review from yourself.

**Comment by nedley** [comment]
> For this PR here, I would prefer to just move the existing data files into the right place, rather than also change the data, especially from a pending PR.
> 
> Also, I don't see any "ordering" file in the other PR with this name or contents.

Sigh, there are too many unreviewed PRs in flight… Just do what you have to do.

**Review by nedley — APPROVED** [review]

**Pull request merged**